open_learning_ai_tutor/StratL.py

```diff
@@ -250,30 +250,26 @@ def _single_message_tutor(
         tools,
     )
     model = client.model_name
-    assessor_client = options.get("assessor_client", None)
-    assessor = Assessor(
-        client=assessor_client,
-        assessment_history=assessment_history,
-        new_messages=new_messages,
+    tutor = Tutor(
+        client,
+        tools=tools,
     )
+    assesment_prompt = get_assesment_prompt(
```

Comment:
The plan is to rewrite intermediary and stratL in the next pr - this has the minimum changes to worked with the combined tutor/assessor model